doc/user/content/administration/billing.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Usage & billing"
+description: "Understand the billing model of Materialize, and learn best practices for cost control."
+menu:
+  main:
+    parent: "manage"
+    weight: 50
+---
+
+{{< note >}}
+Access to usage and billing requires **administrator** privileges.
+{{</ note >}}
+
+Materialize determines billing based on your storage and compute usage.
+
+## Invoice
+
+From the [Materialize console](https://console.materialize.com/) (`Admin` >
+`Usage & Billing`), administrators can access their invoice. The invoice
+provides Compute and Storage usage and cost information.
+
+## Storage
+
+In Materialize, storage is roughly proportional to the size of your source
+datasets plus the size of materialized views, with some overhead from
+uncompacted previous values of data and system metrics.
+
+With the exception of Append-only sources, most data in Materialize is
+continually compacted. As such, the total state stored in Materialize tends to
+grow at a rate that is more similar to OLTP databases than traditional cloud
+data warehouses.
+
+## Compute
+
+In Materialize, [clusters](/concepts/clusters/) are pools of compute resources
```

Comment:
Then I think we can put the rest of the text from the Note below here. A really rough sketch:

> You must provision at least one cluster to power your workloads. You can then use that cluster to create the database objects that will provide you with always-fresh results: views with an index and materialized views.
>
> In Materialize, both views with an index and materialized views are incrementally maintained. That means that Materialize performs work on write, or when Materialize ingests new data. Materialize then performs no work on read, or when you `SELECT` from these objects.
>
> Materialize must be always on...